src/core/core.scale.js

```diff
@@ -240,9 +240,18 @@ var Scale = Element.extend({
 		helpers.callback(this.options.beforeUpdate, [this]);
 	},
 
+	/**
+	 * @param {number} maxWidth - the max width in pixels
+	 * @param {number} maxHeight - the max height in pixels
+	 * @param {object} margins - the space between the edge of the scale and edge of the chart
+	 *   This space comes from two sources:
+	 *     - padding - space that's required to show the labels at the edges of the scale
+	 *     - thickness of scales or legends in another orientation
+	 */
 	update: function(maxWidth, maxHeight, margins) {
 		var me = this;
-		var i, ilen, labels, label, ticks, tick;
+		var optionTicks = me.options.ticks;
```

Comment:
These are aliased to `tickOpts`, `ticksOpts` and `optionTicks` (maybe more), in various places. I like `tickOpts` best.

Reply:
Agreed